lib/server.js

```diff
@@ -191,7 +191,7 @@ Server.prototype.close = function () {
   if (this.ws) {
     debug('closing webSocketServer');
     this.ws.close();
-    delete this.ws;
+    // don't delete this.ws because it can be used again if the server starts listening again
```

Comment:
Yeah it's there in order for people not to add it again in the future, and give the reasoning why.
I tend to add this kind of comments since sometimes I forget the reason I removed a certain line and saves me the time in trying it again, but of course it can be removed. It'll be a bit redundant if we manage to test this